### PR TITLE
Create homepage slider

### PR DESCRIPTION
**Pull request by EasileleM**
Add slider block with 3 slides by default.
Slide has product info on it, some background picture and slider controller.
All info should be downloaded from server.
Controller changes slides after timeout.
Slide change animation has smooth transition

### CODE REVIEW

**Comment by EasileleM** [comment]
Do not merge it yet pls.

**Comment by EasileleM** [comment]
Haven't added item fetching yet.

File: client/src/components/slider/contentBlock/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { PureComponent } from 'react';
+import { Transition } from 'react-transition-group';
+
+import { Item } from '../item/index';
+
+export class ContentBlock extends PureComponent {
```
Comment (miffn3) [inline client/src/components/slider/contentBlock/index.js]: what about BlockForKIts? ContentForSliderBlock?

File: client/src/components/slider/imagesBlock/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+function Image() {
+  return (
+    <div className="slider__image-wrapper">
+      <img src={Layer1} className="slider__image" alt="Main item" />
+    </div>
+  )
+}
+
+export class Images extends PureComponent {
```
Comment (miffn3) [inline client/src/components/slider/imagesBlock/index.js]: maybe sliderImage?

File: client/src/components/slider/imagesBlock/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  
+
```
Comment (miffn3) [inline client/src/components/slider/imagesBlock/index.js]: too many spaces

File: client/src/components/slider/item/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export class Item extends React.Component {
```
Comment (miffn3) [inline client/src/components/slider/item/index.js]: SliderItem?

File: client/src/style/_fonts.scss [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,400i,700,700i&display=swap');
+
+$primary-font: 'Open Sans', sans-serif;
+
+$font-size-extremely-small: 12px;
```
Comment (miffn3) [inline client/src/style/_fonts.scss]: ???

File: client/src/components/slider/item/index.js (hunk 4 above)
Comment (EasileleM) [reply client/src/components/slider/item/index.js]: they are incapsulated into slider component

**Pull request merged**